<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Agenda::UserCalendarEditorWidget</class>
 <widget class="QWidget" name="Agenda::UserCalendarEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Description</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1" colspan="2">
       <widget class="QTextEdit" name="description">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Icon</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="iconLabel">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="selectIconButton">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Label</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="calendarLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Default event duration</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="defaultDuration">
         <property name="suffix">
          <string> minutes</string>
         </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>2000</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Default location</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="defaultLocation"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Password</string>
         </property>
         <property name="indent">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="password"/>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QCheckBox" name="isDefaultCheck">
         <property name="text">
          <string>Agenda is the default agenda</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="isPrivateCheck">
         <property name="text">
          <string>Agenda is private</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDelegates">
      <attribute name="title">
       <string>Shared</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item row="0" column="0">
        <widget class="Agenda::Internal::UserCalendarDelegatesMapperWidget" name="userCalendarDelegatesWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAvailabilities">
      <attribute name="title">
       <string>Availabilities</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item row="0" column="0" rowspan="4">
        <widget class="QTreeView" name="availabilityView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="addAvailability">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="clearAvail">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="removeAvail">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Agenda::Internal::UserCalendarDelegatesMapperWidget</class>
   <extends>QWidget</extends>
   <header location="global">agendaplugin/usercalendardelegatesmapper.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
